src/panels/lovelace/common/call-service.ts

```diff
@@ -0,0 +1,8 @@
+import { HomeAssistant } from "../../../types";
+
+export const callService = (config: any, hass: HomeAssistant): void => {
```

Comment:
The call is indeed a bit weird, but you can derive from the usage of this object what we expect:

```ts
interface CallServiceConfig {
  service: string;
  entity: string;
  service_data?: { [key: string]: any };
}
```